<?xml version="1.0"?>
<launch>

  <arg name="csv_path_accel_map" default="$(find raw_vehicle_cmd_converter)/data/default/accel_map.csv"/>
  <arg name="csv_path_brake_map" default="$(find raw_vehicle_cmd_converter)/data/default/brake_map.csv"/>

  <arg name="max_throttle" default="0.4"/>
  <arg name="max_brake" default="0.8"/>

  <node pkg="raw_vehicle_cmd_converter" type="raw_vehicle_cmd_converter_node" name="raw_vehicle_cmd_converter" output="screen">
    <param name="csv_path_accel_map" value="$(arg csv_path_accel_map)" />
    <param name="csv_path_brake_map" value="$(arg csv_path_brake_map)" />
    <param name="max_throttle" value="$(arg max_throttle)" />
    <param name="max_brake" value="$(arg max_brake)" />
  </node>
</launch>
